ui/src/app/utils/access-utils.tsx

```diff
@@ -54,20 +51,24 @@ export const redirectToTwoFactorSetup = (): void => {
   window.open(getTwoFactorSetupUrl(), '_blank');
 };
 
-const redirectToNiH = (): void => {
+export const NIH_CALLBACK_PATH = '/nih-callback';
```

Comment:
exported so signed-in-app-routing can also refer to this.

Not our usual pattern so LMKWYT.

Reply:
Hmm, seems like the logical extension of this is to factor out constants for any route that we create an internal link for in the application. Is this the proposal with this pattern?

Reply:
Yes.  I'm not too familiar with best practices in this area, so I don't feel strongly about it.